### DISCUSSION TITLE
Document generator?

### DISCUSSION THREAD

**Discussion by ghost** [objeck/objeck-lang#102] [General]
There is a similar question not yet answered: https://github.com/objeck/objeck-lang/discussions/100

I saw on comments there are `@param` and `@return`. So it seems Objeck is already support generating document from comments?

https://github.com/objeck/objeck-lang/blob/master/core/compiler/lib_src/gtk3.obs

**Comment by objeck** [comment]
Yes, it does. Here is the current [API documentation](https://www.objeck.org/doc/api/index.html).

**Reply by ghost** [reply]
What is the tool used to generate such a documentation and how to use it? Something like Javadoc?

**Comment by objeck** [comment]
[Here's](https://github.com/objeck/objeck-lang/tree/master/programs/doc) the code for documentation tool
